frontend/src/pages/AccountHomePage/AccountHomePage.tsx

```diff
@@ -0,0 +1,88 @@
+import { Button, MaturityRating } from '../../components';
+import Logo from '../../assets/icons/netflix-logo.svg?react';
+import Replay from '../../assets/icons/replay-icon.svg?react';
+import heroUrl from '../../assets/images/hero.webp';
+
+import styles from './AccountHomePage.module.css';
+import { ShowsCarousel } from '../../components/ShowsCarousel';
+import { mockShows } from '../../mock/mockShows';
+import { useEffect, useRef, useState } from 'react';
+
+const AccountHomePage = () => {
+  const ref = useRef<HTMLDivElement>(null);
+  const [leftOffset, setLeftOffset] = useState(0);
+
+  useEffect(() => {
+    const el = ref.current;
+    if (!el) return;
+
+    const updateLeftOffset = () => {
```

Comment:
Sorry, I don't get the idea of this code, could you explain the problem you were trying to solve?